src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java

```diff
@@ -205,14 +206,11 @@ private void processResponse(ContainerRequestContext containerRequestContext,
                                ContainerResponseContext containerResponseContext) {
     if (tracer != null) {
       try {
-        Scope scope = (Scope) containerRequestContext.getProperty(PROPERTY_NAME);
-        if (scope != null) {
-          decorateResponse(containerResponseContext, scope.span());
-          scope.close();
-          scope.span().finish();
-        }
+        Span span = (Span) containerRequestContext.getProperty(PROPERTY_NAME);
+          decorateResponse(containerResponseContext, span);
+          span.finish();
```

Comment:
we should also close the scope, so i think we should pass along both scope and span in containerRequestContext

Reply:
also, nit: extra indent

Reply:
fixed, thx!